engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/history/batch/CleanableHistoricBatchReportResultDto.ftl

```diff
@@ -0,0 +1,35 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/batch/get-cleanable-batch-report/index.html -->
+<#macro dto_macro docsUrl="">
+<@lib.dto >
+    
+    <@lib.property
+        name = "batchType"
+        type = "string"
+        desc = "The type of the batch operation."
+    />
+    
+    <@lib.property
+        name = "historyTimeToLive"
+        type = "integer"
+        format = "int32"
+        desc = "The history time to live of the batch operation."
+    />
+    
+    <@lib.property
+        name = "finishedBatchCount"
+        type = "integer"
+        format = "int64"
+        desc = "The count of the finished batch operations."
+    />
+    
+    <@lib.property
+        name = "cleanableBatchCount"
```

Comment:
The property in the Java code is named `cleanableBatchesCount` instead. I do not see any code that would change the name so I assume the REST docs are wrong? Maybe you can double-check that again 😃 
```suggestion
        name = "cleanableBatchesCount"
```